lib/compressDracoMeshes.js

```diff
@@ -246,6 +250,8 @@ function addCompressionExtensionToPrimitive(gltf, primitive, attributeToId, enco
         bufferView: bufferViewId,
         attributes: attributeToId
     };
+
+    gltf = replaceWithDecompressedPrimitive(gltf, primitive, encodedPointsCount, encodedFacesCount, primitive.extensions.KHR_draco_mesh_compression);
```

Comment:
Small thing, but instead of passing in both `primitive` and `primitive.extensions.KHR_draco_mesh_compression`, maybe just pass in `primitive` and get the extension in the function itself.

Reply:
Done.